docs/support/cardano.md

```diff
@@ -50,6 +50,22 @@ If you query an address and its assets, we will display just those assets that a
 If you want to list all assets of an account, use call in the accounts category, just read the sidenote.
 :::
 
+## Pool.pm uses fingerprints for querying assets, why don't you too?
+
+That's right, we currently only allow only `policy_id+hex_encoded_asset_name` as valid input. Why?
+
+TL;DR: It's the safest option.
+
+`Fingerprint`, introduced in [CIP-14](https://cips.cardano.org/cips/cip14/), is a tradeoff between readability and security.
+Fingerprints are shorter hashes. By definition, hashes are not unique. Therefore, it is possible to create two different assets with the same hash (`fingerprint`).
```

Comment:
```suggestion
Fingerprints are shorter hashes. By definition, hashes are not unique.
```

basically the same as
"...it is not impossible for a malicious attacker to forge an asset with identical fingerprint to some other asset."